planning/behavior_path_planner/autoware_behavior_path_lane_change_module/src/scene.cpp

```diff
@@ -1403,24 +1401,31 @@ bool NormalLaneChange::hasEnoughLengthToTrafficLight(
          dist_to_next_traffic_light_from_lc_start_pose >= path.info.length.lane_changing;
 }
 
-bool NormalLaneChange::getLaneChangePaths(
-  const lanelet::ConstLanelets & current_lanes, const lanelet::ConstLanelets & target_lanes,
-  Direction direction, const bool is_stuck, LaneChangePaths * candidate_paths) const
+bool NormalLaneChange::getLaneChangePaths(LaneChangePaths * candidate_paths) const
```

Comment:
It would be nice to get rid of raw pointers if possible. Here I think `candidate_paths` can be passed by reference.